demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -72,7 +76,7 @@ def __init__(
         self.lines_inserted_at_code_start = 0
         self.config = configuration or Configuration()
         self.auto_create_dir = not no_auto_create_dir
-        self.yml_data = get_yaml(self.input)
+        self.yml_data = loaded_data if loaded_data is not None else get_yaml(self.input)
```

Comment:
```suggestion
        self.yml_data = loaded_data if loaded_data else get_yaml(self.input)
```
why not like this?

Reply:
Because I wanted to avoid re-loading the file in an edge case where `loaded_data` is an empty dict (`{}`).